source/crud/tests/unified/bulkWrite-comment.yml

```diff
@@ -148,6 +137,7 @@ tests:
                 delete: *collection_name
                 deletes:
                   - q: *deleteOne_filter
+                    limit: 1
```

Comment:
This is good. All spec tests should explicitly specify `1` or `0` for `limit` on `delete` commands.